projects/ccf-eui/src/app/shared/components/spatial-search-config/spatial-search-config.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="spatial-search-container">
```

Comment:
Don't use top level container div. Place styles directly in the `:host` selector instead